templates/app_payload_template.jinja2

```diff
@@ -1,12 +1,19 @@
 {
   "client_id": "{{client_id}}",
+  {{user_id}}
   "timestamp_micros": {{event_timestamp}},
-  "nonPersonalizedAds": false,
+  "consent": {
```

Comment:
@kingman are those changes correct? can you point the source information you used to build this part of the payload, pls.

Reply:
https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#payload_post_body
from the doc:
```
Obsolete: Use the ad_personalization field of [consent](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#payload_consent) instead of this field. Google Analytics accepts either field, but ad_personalization is recommended.
```